multiapi-engine/src/main/java/com/sngular/api/generator/plugin/commonApi/MapperUtilCommon.java

```diff
@@ -0,0 +1,199 @@
+/*
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  * License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.sngular.api.generator.plugin.commonApi;
+
+import static com.sngular.api.generator.plugin.commonApi.Constants.*;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public abstract class MapperUtilCommon {
+
+  public MapperUtilCommon() {}
+
+  public  static String getSimpleType(final JsonNode schema, final String prefix, final String suffix) {
+    String type;
+    if ((schema.has(TYPE)) && (!NULL.equalsIgnoreCase(schema.get(TYPE).toString()))){
+      type = schema.get(TYPE).textValue();
+      String format = null;
+      if (STRING.equalsIgnoreCase(type)) type = STRING;
+      if (schema.has(FORMAT)) format = schema.get(FORMAT).textValue();
+
+      if (NUMBER.equalsIgnoreCase(type)) {
+        if (FLOAT.equalsIgnoreCase(format)) {
+          type = FLOAT;
+        } else if (DOUBLE.equalsIgnoreCase(format)) {
+          type = DOUBLE;
+        } else {
+          type = BIG_DECIMAL;
+        }
+      } else if (INTEGER.equalsIgnoreCase(type)) {
+        if (INT_64.equalsIgnoreCase(format)) {
+          type = LONG;
+        } else {
+          type = INTEGER;
+        }
+      }
+
+    } else  {
+      type = getPojoName(getRefClass(schema), prefix, suffix);
+    }
+    return StringUtils.capitalize(type);
+  }
+
+  public static String getTextValue(JsonNode node, String field){
+    try{
+      if(checkTextValueNotNull(node.get(field)))
+        return node.get(field).textValue();
+    }
+    catch (StackOverflowError e){
+      return null;
+    }
+    return null;
+  }
```

Comment:
I was thinking... what would change if we do something like:
```
catch (StackOverflowError e){}
finally {
      return null;
    }
}
```
To avoid duplicating `return null;`